pro/src/apiClient/v1/models/CollectiveBookingCancellationReasons.ts

```diff
@@ -15,4 +15,10 @@ export enum CollectiveBookingCancellationReasons {
   PUBLIC_API = 'PUBLIC_API',
   FINANCE_INCIDENT = 'FINANCE_INCIDENT',
   BACKOFFICE = 'BACKOFFICE',
+  BACKOFFICE_EVENT_CANCELLED = 'BACKOFFICE_EVENT_CANCELLED',
+  BACKOFFICE_SURBOOKING = 'BACKOFFICE_SURBOOKING',
+  BACKOFFICE_BENEFICIARY_REQUEST = 'BACKOFFICE_BENEFICIARY_REQUEST',
+  BACKOFFICE_OFFER_MODIFIED = 'BACKOFFICE_OFFER_MODIFIED',
+  BACKOFFICE_OFFER_ERROR = 'BACKOFFICE_OFFER_ERROR',
+  OFFERER_CONNECT_AS = 'OFFERER_CONNECT_AS',
```

Comment:
Je vois un `do not edit` en haut du fichier, tu l'as modifié à la main ?

Reply:
oui

Reply:
Bah fallait pas mdr